ReactWindows/ReactNative.Net46/UIManager/BaseViewManager.cs

```diff
@@ -20,34 +20,52 @@ namespace ReactNative.UIManager
     /// <typeparam name="TFrameworkElement">Type of framework element.</typeparam>
     /// <typeparam name="TLayoutShadowNode">Type of shadow node.</typeparam>
     public abstract class BaseViewManager<TFrameworkElement, TLayoutShadowNode> :
-            ViewManager<TFrameworkElement, TLayoutShadowNode>
+            ViewManager<TFrameworkElement, TLayoutShadowNode>,
+            ITransformControl
         where TFrameworkElement : FrameworkElement
         where TLayoutShadowNode : LayoutShadowNode
     {
-        private readonly IDictionary<TFrameworkElement, Action<TFrameworkElement, Dimensions>> _transforms =
-            new Dictionary<TFrameworkElement, Action<TFrameworkElement, Dimensions>>();
+        private readonly IDictionary<TFrameworkElement, Tuple<Action<TFrameworkElement, Dimensions>, JArray>> _transforms =
```

Comment:
>Tuple<Action<TFrameworkElement, Dimensions>, JArray> [](start = 56, length = 52)

We can make this a bit more concise and readable below (e.g., `Item1` and `Item2` is not very informative) if we use a class to contain this data instead.

Reply:
Ok, I'll try. Tuple is basically useless by this definition, they should allow some inline field naming syntax :).

---
In reply to: [159513233](https://github.com/Microsoft/react-native-windows/pull/1546#discussion_r159513233) [](ancestors = 159513233)

Reply:
Fair point. It's really a nitpick anyway. Doesn't really matter, I'd probably be less concerned if the second TParam wasn't `JArray`... I sort of forgot what the `JArray` was when I was reading below :grinning: 

---
In reply to: [159516260](https://github.com/Microsoft/react-native-windows/pull/1546#discussion_r159516260) [](ancestors = 159516260,159513233)